website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -527,13 +527,37 @@ The first handler takes a `NewGravatar` event and creates a new `Gravatar` entit
 
 The second handler tries to load the existing `Gravatar` from the Graph Node store. If it does not exist yet, it is created on-demand. The entity is then updated to match the new event parameters before it is saved back to the store using `gravatar.save()`.
 
-### Recommended IDs for Creating New Entities
+### Using Bytes as IDs
```

Comment:
I'd keep the original title. For a new dev, I think this does not make sense. They simply want to learn "The recommended way for IDs"